.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = Object.assign({}, sharedConfigs, {
         '@typescript-eslint/ban-types': 'off',
         '@typescript-eslint/no-inferrable-types': 'off',
         '@typescript-eslint/no-this-alias': 'off',
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
what's this for? I've removed those lines you added and lint passed without issues.

Reply:
it passes but generates 600+ warnings which makes it pretty hard to find anything in the output

Reply:
FYI eslint has the `--quiet` flag that hides all warnings so only errors are displayed.

Reply:
also wanted to avoid it on github actions, I don't see those warnings useful anyhow if that passes the CI

Reply:
yeah, I agree with disabling it.